modules/ROOT/pages/aggregator/aggregators-module-reference.adoc

```diff
@@ -265,6 +265,7 @@ Listeners are only able to reference aggregators that are inside a flow. If an a
 
 Once the aggregator that is referenced by the listener completes an aggregation, the listener is triggered with a list of all the elements. Though the aggregation listener can be used for any kind of aggregator, it is important for time-driven (Async) aggregations. Such aggregations are triggered asynchronously, so they do not execute an aggregator route and can only reach components in flows with an aggregator listener as the source.
 
+Using an aggregator listener creates a new flow. The context of the flow where the aggregator exists is not accessible from the new flow. Therefore, the new flow cannot access variables from the aggregrator's flow.
```

Comment:
The aggregator listener does not create a new flow. In any case, it should be: 
"Since the aggregator listener is a Source, it should be added as such in a different flow where the actual Aggregator is. The context of the flow where the aggregator...."